### PR TITLE
Review workflow & Nomenclature

### PR DESCRIPTION
**Pull request by soujash-mandal**
1. if all reviewer reviewed then only moved to review- completed.
move to approved button  for doc owner only works if all of the reviewers have reviewed.
When Last Reviewer Reviews the doc status will change to Reviewed .

To Check This functionality Only Check files  
# sidebar.hbs And sidebar.ts.

2. Changed the names of approve and WIP 
3. removed longName from template

### CODE REVIEW

File: config.hcl [hunk 1]
```diff
@@ -48,7 +48,7 @@ document_types {
   }
 
   document_type "PRD" {
-    long_name   = "Product Requirements"
+    // long_name   = "Product Requirements"
     description = "Create a Product Requirements Document to summarize a problem statement and outline a phased approach to addressing the problem."
```
Comment (yashcode00) [inline config.hcl:52]: remove these comments completely

File: internal/api/custom_templates.go [hunk 2]
```diff
@@ -227,15 +227,15 @@ func TemplateUpdateDeleteHandler(
...
 				DocId string `json:"docId"`
 			}
 			
 			updateObj := template{
 				ObjectID: ObjectID,
 				TemplateName: req.TemplateName,
-				LongName: req.LongName,
+				// LongName: req.LongName,
 				Description: req.Description,
 				DocId: req.DocId,
```
Comment (yashcode00) [inline internal/api/custom_templates.go:240]: remove comented part

File: internal/cmd/commands/server/server.go
Comment (yashcode00) [inline internal/cmd/commands/server/server.go:1]: remove commented long name comppletely

File: internal/config/config.go
Comment (yashcode00) [inline internal/config/config.go:1]: remove commented long name comppletely

File: web/app/components/new/template-form.ts [hunk 3]
```diff
@@ -115,9 +115,21 @@ export default class NewDocFormComponent extends Component<NewTemplateFormCompon
   /**
    * Sets `formRequirementsMet` and conditionally validates the form.
    */
+  // private maybeValidate() {
+  //   this.docId=this.extractDocId(this.docId)
+  //   if (this.templateName && this.longName && this.docId.length==44) {
+  //     this.formRequirementsMet = true;
+  //   } else {
+  //     this.formRequirementsMet = false;
```
Comment (yashcode00) [inline web/app/components/new/template-form.ts:123]: remove commented part

**Review by yashcode00 — APPROVED** [review]

File: web/app/components/new/template-form.ts (hunk 3 above)
Comment (soujash-mandal) [reply web/app/components/new/template-form.ts]: ok

**Pull request merged**